addons/account/models/product.py

```diff
@@ -64,7 +64,7 @@ def get_product_accounts(self, fiscal_pos=None):
     def _compute_tax_string(self):
         for record in self:
             currency = record.currency_id
-            res = record.taxes_id.compute_all(record.list_price)
+            res = record.taxes_id.compute_all(record.list_price, product=record)
```

Comment:
You also need to include `quantity=1` and `partner=self.env['res.partner']` (`company` and `price_unit` already given)

https://github.com/odoo/odoo/blob/6d206e74dca14cf1f783906e6f30efa6664b7120/addons/account_tax_python/models/account_tax.py#L49

Reply:
Ok for ```partner``` but  ```quantity``` is by default set to 1.0